test/Telegram.Bot.Tests.Integ/Other/LeaveChatTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Threading.Tasks;
+using Telegram.Bot.Tests.Integ.Framework;
+using Telegram.Bot.Tests.Integ.Framework.Fixtures;
+using Xunit;
+
+namespace Telegram.Bot.Tests.Integ.Other
+{
+    [Collection(Constants.TestCollections.LeaveChat)]
+    [TestCaseOrderer(Constants.TestCaseOrderer2, Constants.AssemblyName)]
+    public class LeaveChatTests : IClassFixture<LeaveChatTests.Fixture>
+    {
+        private ITelegramBotClient BotClient => _fixture.BotClient;
+
+        private readonly Fixture _classFixture;
+
+        private readonly TestsFixture _fixture;
+
+        public LeaveChatTests(TestsFixture fixture, Fixture classFixture)
+        {
+            _fixture = fixture;
+            _classFixture = classFixture;
+        }
+
+        [OrderedFact(DisplayName = FactTitles.ShouldLeaveChat,
+            Skip = "Bot should stay in chat for other the test cases")]
```

Comment:
Bot can ask the user to create a group and add it to the group

Reply:
that requires too much interaction by user. I was trying to keep it as automated as possible. It's a very simple method though.